src/convertToObject.js

```diff
@@ -5,8 +5,21 @@
  *
  * @return {object}
  */
+
 function convertToObject(sourceString) {
-  // write your code here
+  const stylesArray = sourceString.split(';');
+
+  const stylesObject = {};
+
+  stylesArray.forEach((style) => {
+    const [key, value] = style.split(':').map((item) => item.trim());
+
+    if (key && value) {
+      stylesObject[key.trim()] = value.trim();
```

Comment:
Fix it
<img width="852" alt="image" src="https://github.com/user-attachments/assets/77b87e41-1a38-47d1-8636-266c5f16b420">
```suggestion
      stylesObject[key] = value;
```